### ISSUE TITLE
Thought Leadership Singles (COMMENT FOR ALL SINGLES) https://www.geiconsultants.com/thought_leadership/geis-cam-davis-interviewed-by-wisconsin-public-radio/

### ISSUE THREAD

**Issue by kristenchapline-design** [kristenchapline-design/GEI#52]
Links must have discernible text

Element location
a[aria-title="share\ on\ Facebook"]
a[aria-title="share\ on\ Twitter"]
a[aria-title="share\ on\ Linkedin"]

To solve this violation, you need to:
Fix at least one (1) of these issues:
Element does not have text that is visible to screen readers

aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Element's default semantics were not overridden with role="presentation"

Element's default semantics were not overridden with role="none"

And

Fix the following:
Element is in tab order and does not have accessible text

**Comment by kristenchapline-design** [comment]
issue still appears for linkedin

**Comment by kristenchapline-design** [comment]
still coming up as an issue on the insight singles


Element location
a[aria-title="share\ on\ Linkedin"]

Element source
<a class="linkedin" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://www.geiconsultants.com/thought_leadership/geis-cam-davis-interviewed-by-wisconsin-public-radio/&amp;title=GEI’s%20Cam%20Davis%20Interviewed%20by%20Wisconsin%20Public%20Radio" target="_blank" aria-title="share on Linkedin">
